learning_resources/filters.py

```diff
@@ -106,3 +106,69 @@ def filter_sortby(self, queryset, _, value):
     class Meta:
         model = LearningResource
         fields = ["professional"]
+
+
+class ContentFileFilter(FilterSet):
+    """ContentFile filter"""
+
+    run_id = CharFilter(
+        label="The id of the learning resource run the content file belongs to",
+        field_name="run_id",
+        lookup_expr="exact",
+    )
+
+    learning_resource_id = CharFilter(
```

Comment:
If I pass a non-numeric value for `run_id` or `learning_resource_id` (like `?run_id=1a`), a `ValueError` exception is thrown. 

 Can you change the class for these two to `NumericFilter` instead?  That way a formatted error message will be returned instead:

![Screenshot 2023-12-19 at 9 46 09 AM](https://github.com/mitodl/mit-open/assets/187676/db98a759-8fb6-4f4a-b4a2-4cde8c4ca565)